racktables/find_pingable_machines.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+found_machines=0
+while read fqdn; do
```

Comment:
shellcheck would probably suggests

```suggestion
while read -r fqdn; do
```

Reply:
might make sense if machine names contain funny characters :)